devel/ihaskell/Portfile

```diff
@@ -4,10 +4,13 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           haskell_stack 1.0
 
-github.setup        gibiansky IHaskell e07b198f07d7dc7a12863f8507439ae3834947e4
+github.setup        gibiansky IHaskell 0f1262d3e710518fd734fbda6f2eba33e476836b
 name                [string tolower ${github.project}]
-version             2021.01.28
+# version from https://github.com/gibiansky/IHaskell/blob/master/ihaskell.cabal
+# with commit date appended
+version             0.10.1.2.20210228
```

Comment:
I think we should either use an official upstream version, or a date, but not a combination of both. A date is used when upstream does not do proper releases, or if the port specifically is designed to provide a ‘devel’ snapshot version.

Reply:
Done thanks.